services/datcore-adapter/src/simcore_service_datcore_adapter/api/routes/files.py

```diff
@@ -53,3 +54,27 @@ async def delete_file(
         api_secret=x_datcore_api_secret,
         obj_id=file_id,
     )
+
+
+@router.get(
+    "/packages/{package_id}/files",
```

Comment:
so if I get that correctly, you always return 1 file right?
from the interface I would  multiple files. Is this never the case? do you always get only 1 file or there cannot be multiple ones?

Reply:
I am also not totally sure about this point. I also thought that the interface would return more than 1 file, but I never saw that to be the case. 

"never"=I tried a couple of different {package_ids} and those always return a list with 1 file. 😅 

 That's why I've added that (wrong, I need to fix it) assert [here](https://github.com/ITISFoundation/osparc-simcore/pull/5357/files/cc59064a16d5a09c5eddd701410794bd85c4667a#diff-977fdbce36c9aece99c231abacfb9639dc4e2cb346500a92eea2b001a1103259).

Happy to discuss if you think we should handle this better.